### DISCUSSION TITLE
Support one-click unsubscribe

### DISCUSSION THREAD

**Discussion by SydneyUni-Jim** [pkp/pkp-lib#10171] [Proposals]
- Support adding List-Unsubscribe ([RFC 2369](https://www.ietf.org/rfc/rfc2369.txt)) and List-Unsubscribe-Post ([RFC 8058](https://www.rfc-editor.org/rfc/rfc8058)) to emails.
- Support suffixing unsubscribe links to the footers of emails.
- Provide a landing page to allowing unsubscribing from all context or particular contexts. Unsubscribe would remove all least the Reader role, perhaps all roles.
- Have both the unsubscribe link and the List-Unsubscribe header take end-users to the unsubscribe landing page.

### Background

Large mail providers — such as Gmail — are starting to enforce one-click unsubscribe for "bulk" senders.

### References

- [New Gmail protections for a safer, less spammy inbox](https://blog.google/products/gmail/gmail-security-authentication-spam-protection/)
- [Email sender guidelines FAQ](https://support.google.com/a/answer/14229414)
- [Using one-click unsubscribe with Amazon SES](https://aws.amazon.com/blogs/messaging-and-targeting/using-one-click-unsubscribe-with-amazon-ses/)

**Comment by asmecher** [comment]
@SydneyUni-Jim, see https://github.com/pkp/pkp-lib/issues/6627, which has been include in OJS/OMP/OPS 3.4.0-5. [ed: Also [#7874](https://github.com/pkp/pkp-lib/issues/7874), which was included in 3.4.0-0.)

**Reply by SydneyUni-Jim** [reply]
Thank you @asmecher. We're still on 3.3 LTS. I look forward to being able to use this when 3.5 LTS is released.
